scripts/create_gw5000_database.py

```diff
@@ -0,0 +1,142 @@
+import pandas as pd
+import re
+from mendeleev import element
+from collections import Counter
+from tqdm import tqdm
+from dateutil import tz
+from datetime import datetime
+
+def convert_smiles_to_molecular_formula(inchi: str) -> str:
```

Comment:
Is this converting SMILES to molecular formula, or converting InChI to molecular formula?

Reply:
Corrected. It was supposed to be inchi to molecular formula. I had forgotten to update the function name.